components/unarchive_channel_modal/index.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import {unarchiveChannel} from 'mattermost-redux/actions/channels';
+import {getConfig} from 'mattermost-redux/selectors/entities/general';
+
+import UnarchiveChannelModal from './unarchive_channel_modal.jsx';
+
+function mapStateToProps(state) {
+    const config = getConfig(state);
+
+    return {
+        canViewArchivedChannels: config.ExperimentalViewArchivedChannels === 'true',
```

Comment:
If the extra translation string is removed, this can also be removed.

Reply:
@hmhealey   I see what youre saying.  i removed all of them now. and updated snap